maven-plugin/src/main/resources/openmrs-distro-3.0.properties

```diff
@@ -0,0 +1,61 @@
+name=Ref 3.x distro
+version=3.0.0
+war.openmrs=2.6.0
+config.referenceapplication-config=3.0.0-alpha.7
+omod.initializer=2.5.1
```

Comment:
Didn't we agree not to embed a distro.properties?

Reply:
Yeah. Sorry, I'll fix that in the next commit.

Reply:
Did you also see the failing tests?

Reply:
Yes, I committed a fix it.

Reply:
Yeah, I tried it a few times. Sometimes when building the O3, I got the following error after the initial setup, but restarting the server fixed it for me.

`An error has occurred!
The following error happened somewhere on this page:
javax.servlet.jsp.JspTagException: Theme 'green': No message found under code 'favicon' for locale 'en_US'.
`
Logs: https://pastebin.com/BqycyZn4

Reply:
Are you able to consistently reproduce this error every time you run the initial setup?

Reply:
No. I tried building the O3 again and did not get that error. I only got that error only like two times.

Reply:
I found this thread about that error: https://talk.openmrs.org/t/theme-green-no-message-found-under-code-favicon-for-locale-en-gb/30836

Reply:
Putting that error aside, is the SDK now working as expected with your changes in this pull request? If yes, can you give me the step by step instructions that i should use to try it out locally?

Reply:
yeah.
1.  First set up the server with `mvn org.openmrs.maven.plugins:openmrs-sdk-maven-plugin:4.6.0-SNAPSHOT:setup`
2. Select the O3 Reference Application option
3. Run the server with `mvn org.openmrs.maven.plugins:openmrs-sdk-maven-plugin:4.6.0-SNAPSHOT:run`

You can also setup the server by passing the distro.properties file as an argument.

`mvn org.openmrs.maven.plugins:openmrs-sdk-maven-plugin:4.6.0-SNAPSHOT:setup -Ddistro=distro.properties`

Reply:
@dkayiwa are you getting any errors??

Reply:
Which version of O3 modules do i get when i do not supply the distro.properties file?

Reply:
When you don't supply a distro.properties file SDK uses the distro.properties in the resource folder. It contains the 'next' version of the esm modules.